tools/aliaslookup/aliaslookup.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+"""Utility to look up OSV records by alias.
+"""
+
+from google.cloud import datastore
+
+import sys
+import fileinput
+import argparse
+
+
+def lookup_by_aliases(client: datastore.Client,
+                      identifiers: list[str],
+                      verbose=False) -> datastore.query.Iterator:
+  """Look up OSV records by alias.
+
+  Args:
+    client: a datastore.Client object.
+    identifiers: a list of strings being the aliases to look up.
+    verbose: a boolean whether to emit more verbose processing information.
+
+  Returns:
+    a datastore.query.Iterator
+  """
+  query = client.query(kind="Bug")
+  # the "IN" operator only supports 30 identifiers as a maximum, so we need to
+  # batch into multiple a composite OR query
+  # This still caps out at 100 total filters, sadly
+  query.add_filter(
+      filter=datastore.query.PropertyFilter("aliases", "IN", identifiers))
```

Comment:
Could we instead use our ndb client for consistency with the rest of our codebase? 

We also make our "osv" library available as a pypi package: https://pypi.org/project/osv/ where we can pull in the necessary models.

Reply:
I could, but not in this PR.

I went with the datastore client library because I was cribbing from the previous withdrawal code I'd written, and realised the ndb client was the newer thing as part of needing to unpin it, to get "IN" operator support, and started trying to switch over to it, but got into more trouble than I wanted to deal with trying to get imports to work successfully for local execution.

This tool, as-is, meets my immediate needs.